docs/search/activity_log_search_reference/logged_at_criterion.md

```diff
@@ -0,0 +1,26 @@
+# LoggedAt Criterion
+
+The [`LoggedAtCriterion` Activity Log Criterion](https://github.com/ibexa/activity-log/blob/main/src/contracts/Values/ActivityLog/Criterion/LoggetAtCriterion.php)
+matches activity log group having a log entry created before or after a given date time.
+
+## Arguments
+
+- `dateTime` - a [`DateTimeInterface`](https://www.php.net/manual/en/class.datetimeinterface.php) object, like [`DateTime`](https://www.php.net/manual/en/class.datetime.php).
+- `comparison` - a comparison sign as a string. Available signs can be found as constant in the `LoggedAtCriterion` class itself.
+
+| Comparison            | Value | Constant                 |
+|-----------------------|-------|--------------------------|
+| Equal                 | `=`   | `LoggedAtCriterion::EQ`  |
+| Not equal             | `<>`  | `LoggedAtCriterion::NEQ` |
+| Less than             | `<`   | `LoggedAtCriterion::LT`  |
+| Less than or equal    | `<=`  | `LoggedAtCriterion::LTE` |
+| Greater than          | `>`   | `LoggedAtCriterion::GT`  |
+| Greater than or equal | `>=`  | `LoggedAtCriterion::GTE` |
+
+## Example
```

Comment:
I'd expect REST examples for all of these as well :/

Reply:
I effectively miss examples in https://ez-systems-developer-documentation--2157.com.readthedocs.build/en/2157/api/rest_api/rest_api_reference/rest_api_reference.html#monitoring-activity-list-of-activity-log-groups

Reply:
Similar table added to REST Ref (#2157), see https://ez-systems-developer-documentation--2157.com.readthedocs.build/en/2157/api/rest_api/rest_api_reference/rest_api_reference.html#monitoring-activity-list-of-activity-log-groups